### PR TITLE
Entry point events

### PR DESCRIPTION
**Pull request by TheBurchLog**
This should fix the plugins from starting before the HTTP Entry point

### CODE REVIEW

File: src/app/beer_garden/api/http/__init__.py [hunk 1]
```diff
@@ -124,7 +126,9 @@ async def shutdown():
     # This will almost definitely not be published to the websocket, because it would
     # need to make it up to the main process and back down into this process. We just
     # publish this here in case the main process is looking for it.
-    publish(Event(name=Events.ENTRY_STOPPED.name))
+    publish(
+        Event(name=Events.ENTRY_STOPPED.name, metadata={"entry_point_type": "STOMP"})
```
Comment (hazmat345) [inline src/app/beer_garden/api/http/__init__.py:130]: This should be HTTP

**Review by hazmat345 — CHANGES_REQUESTED** [review]
Do we not publish an entry stopped for the stomp entry point?

**Review by west270 — COMMENTED** [review]
LGTM

**Comment by TheBurchLog** [comment]
Just added the stop metadata for STOMP, didn't realize that was thrown in the Stomp Manager.

**Review by hazmat345 — APPROVED** [review]
👍🏼

**Pull request merged**